docs/usage/templates.md

```diff
@@ -30,3 +30,23 @@ If you want to print pretty JSON with Handlebars you can use the built-in functi
 `{{{stringToPrettyJSON myvar}}}`
 
 In the example above `myvar` is a variable/field, that contains valid JSON.
+
+### encodeURIComponent
+
+If you want to convert a string to a valid URI, use the built-in function `encodeURIComponent` like this:
+
+`{{{encodeURIComponent myvar}}}`
+
+In the example above `myvar` is the string you want to transform into a valid URI.
+
+Read the [MDN Web Docs, encodeURIComponent()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) to learn more.
```

Comment:
I've added links to the MDN Web Docs, but maybe these should be links to the appropriate documentation from Handlebars themselves (if they are available)?

Reply:
There's nothing from handlebars